src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ContestedCaseOrderService.java

```diff
@@ -27,7 +40,41 @@ public void printAndMailGeneralOrderToParties(CaseDetails caseDetails, String au
         }
     }
 
+    public void printAndMailHearingDocuments(CaseDetails caseDetails, String authorisationToken) {
+        if (isContestedPaperApplication(caseDetails)) {
```

Comment:
if you move this check to the controller then this method won't be updated when we want to use it for consented

Reply:
This endpoint will never be called for consented